routes/api/v1/forecast.js

```diff
@@ -0,0 +1,92 @@
+require('dotenv').config();
+
+var express = require('express');
+var fetch = require('node-fetch');
+
+var User = require('../../../models').User;
+var forecastSerializer = require('../../../serializers/forecast_serializer')
+
+var router = express.Router();
+
+router.get('/', (request, response) => {
```

Comment:
@bplantico What do you think about the implementation of the API calls here?  Is it worth setting up a separate service, or is fetch basically functioning in the same way?  How would you refactor this to be a little cleaner?

Reply:
@alexander-mathieu Nice job getting this working for the time being. This file is definitely doing a lot, so if you're able to figure out how to refactor to abstract the calls to different services (similar to how you did in Rails)
.then( this => {
file will be cleaner and you'll have the ability to use the services in other place if need be
});

I imagine you'll need to make a file for both the Google GeoCoding API call and one for the DarkSky API call, then remember to export from those files and require them in your `forecast.js`

Reply:
Cool man.  I think 'for the time being' is the moral of the story so far.

I'm going to pull this in for now, and make a new task on my project board to refactor the calls.  I'll tag you on the next PR.

Thanks again, let me know what I can review for you.